### PR TITLE
Akshay Gadkari PR

### PR DESCRIPTION
**Pull request by akshay-gadkari**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -4,4 +4,16 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  NavLink
+} from "react-router-dom";
```
Comment (DrewMoody) [inline client/src/index.js]: You only need to import what you use. So for index.js, you only need BrowserRouter as Router. Everything else you can get rid of.

File: client/src/App.js [hunk 2]
```diff
@@ -4,6 +4,13 @@ import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
 
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  NavLink
```
Comment (DrewMoody) [inline client/src/App.js]: Same as first comment. For your app.js, you can get rid of BrowserRouter, Link, and Navlink. Make sure you're checking the warnings in your console during development.

File: client/src/Movies/MovieList.js [hunk 3]
```diff
@@ -33,22 +34,22 @@ export default class MovieList extends Component {
...
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
+    return (
+	<Link to={`/movies/${movie.id}`} className="movie-card">
```
Comment (DrewMoody) [inline client/src/Movies/MovieList.js]: The issues you had with styling are the result of this. A better approach would be to keep that original div with className of movie-card, and then do your link separate.
For example, you can do:
<Link to={`/movies/${movie.id}`} >
<div className='movie-card'>
With that approach, the styling works fine.

File: client/src/App.js [hunk 4]
```diff
@@ -18,12 +25,20 @@ export default class App extends Component {
...
+	    <div>
+              <SavedList list={this.state.savedList} />
+		<Route
+		  exact
+		  path='/'
+		  component={MovieList}
+		  />
+		<Route
+		  path='/movies/:id'
+		  component={Movie}
```
Comment (DrewMoody) [inline client/src/App.js]: This works, but it is better to use render than component. So:
render={props => <Movie {...props} />}
It's technical, but explanation here:
https://stackoverflow.com/questions/48150567/react-router-difference-between-component-and-render
TLDR - component creates new component each time, render does not. Render is a bit faster.

**Comment by akshay-gadkari** [comment]
Thanks! I'll use render for today's project and check for warnings.
